addons/http_routing/models/ir_http.py

```diff
@@ -114,7 +114,9 @@ def unslug(s):
 
 
 def unslug_url(s):
-    """ From /blog/my-super-blog-1" to "blog/1" """
+    """ From /blog/my-super-blog-1" to "blog/1"
+        Note that any query string or anchor will prevent the unslug
```

Comment:
Is not that what you fixed in the first commit? I tested:
```
In [4]: unslug_url('hello/world-2?foo=bar')
Out[4]: 'hello/2'
```
It seems to lose the querystring but not prevent the unslug

Reply:
Indeed, it's what I fixed in the first commit that landed afterward, as I was sad keeping this misbehavior, despite commenting it.
This should have thus been removed as not true anymore, good catch.